src/main/scala/io/iohk/ethereum/network/ForkResolver.scala

```diff
@@ -17,17 +17,17 @@ object ForkResolver {
 
   class EtcForkResolver(blockchainConfig: BlockchainConfig) extends ForkResolver {
```

Comment:
As long as it's clear that in those names `Ethereum` means general Ethereum - all chains, not just Ethereum HF, then I think that's the way to go. As usual I like to call @mcsherrylabs on naming related issues.

Separate task would be useful so as not to inhibit other dependent ETH tasks.